python/mlc_chat/support/max_thread_check.py

```diff
@@ -0,0 +1,14 @@
+"""Helper functions for checking max num thread."""
+
+from tvm.target import Target
+
+
+def check_max_num_threads(target: Target, bdx: int, bdy: int, bdz: int):
+    """Check whether max num threads exceeded given a target."""
+    assert (
+        bdx * bdy * bdz <= target.max_num_threads
+    ), f"{target.kind} max num threads exceeded: {bdx}*{bdy}*{bdz}>{target.max_num_threads}"
+
+    if str(target.kind) != "webgpu":
```

Comment:
Should it be `== "webgpu"`? Also for WebGPU `blockIdx.z` should be 1, which is also be something we could check.

Reply:
Resolved, thank you!